packages/open-truss/src/configuration/engine-v1.tsx

```diff
@@ -62,15 +61,19 @@ export function engineV1(
       }
 
       if (subFrame === undefined) {
-        return Component({ ...props })
```

Comment:
If / when we do need to go back to async components, let's remember this is a valid way to call components and how to fix the bugs we were seeing.